stdlib/test/builtin/test_simd.mojo

```diff
@@ -1545,6 +1545,43 @@ def test_contains():
     assert_false(0 in y or 5 in y)
 
 
+def test_count():
```

Comment:
Sorry for split review, but I don't spot that earlier. "A", "B", "C" etc. should be lowercase to follow naming convention

Reply:
Yes, sorry for the convention on naming things, i'll improve as time goes on :+1: 

Next PR reflect the convention !

Please no sorry for split-reviews, and do as much reviews as needed, 

the review process is what refines things so it is fantastic :+1: 

Last message just suggested @martinvuyk to make code suggestions in one block,

as it is difficult to imagine what the final result of merging all small code suggestions looks.